tests/serve/test_vllm_omni.py

```
@@ -109,12 +109,16 @@ class VLLMOmniConfig(EngineConfig):
-            pytest.mark.xpu_1,
-            pytest.mark.post_merge,
-            pytest.mark.timeout(1200),
-            pytest.mark.skip(
-                reason="Qwen/Qwen-Image requires ~40GB GPU memory, exceeds CI capacity (22GB)"
-            ),
+            pytest.mark.h100,
+            # Temporary: pre_merge so PR CI can validate on the H100 lane in
+            # pr.yaml. Flip back to nightly once CI passes (OPS-7582).
+            pytest.mark.pre_merge,
+            pytest.mark.timeout(1800),
+            # Qwen-Image BF16 weights ~40–42 GiB; with VAE tiling/slicing + 2 GiB
```

Ruff (0.15.21)

[warning] Comment contains ambiguous `–` (EN DASH). Did you mean `-` (HYPHEN-MINUS)?

(RUF003)